src/components/styles/Button.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const BaseButton = styled.div<{ fontSize?: string }>`
+  padding: 0.5rem 1rem;
+  .ant-btn-text {
+    height: auto;
+    font-size: ${(props) => props.fontSize};
+    border: 0px;
+    border-bottom: transparent 2px solid;
+    background-color: #000;
+    color: #fff;
+    padding: 1px;
+    padding-bottom: 0px;
+    transition: border-bottom 0.4s ease-in-out;
+    &:hover {
+      border-bottom: 2px solid #fff;
```

Comment:
We should probably coordinate how we want to implement the underlines. I used a slightly different method that might give us more fine grained control over the style and positioning, but this implementation is much simpler and intuitive.

https://github.com/rhdevs/rhdevs-website/blob/908bbb83f6e1db55269f2284339e9efeb6a29ed5/src/components/styles/NavItem.styled.ts#L16-L21

The issue I could see with this is that the underline will always be the length of the entire div, which may or may not annoying moving forward depending on what situations we will need the underline. @hui444 maybe can advise?

Reply:
yes do update to follow this!

Reply:
dd62687540dc1b09686596594c98118730cff4a2